util.go

```diff
@@ -280,17 +279,17 @@ func (b *Builder) untar(chownOpts *idtools.IDPair, hasher io.Writer) func(tarArc
 func isRegistryBlocked(registry string, sc *types.SystemContext) (bool, error) {
```

Comment:
(Absolutely non-blocking:

BTW the three calls of `isReferenceBlocked` (but not the check in `util.ResolveName`) could probably be completely deleted, c/image now enforced the `.Blocked` option automatically.

Note, though:
- The check happens a bit later, which might result in an user-observable behavior change
- c/image enforces the `.Blocked` option on the _results_ of mirror resolution, not on the _input_ to it; i.e. it is possible to have `example.com` mirrored to `foo.local`, with `example.com` blocked and `foo.local` non-blocked; then references to `example.com/…` are allowed but must only physically contact `foo.local`, not access the primary `example.com` server directly.)